docs/src/main/sphinx/release/release-409.md

```diff
@@ -1,15 +1,22 @@
-# Release 409 (1 Mar 2023)
+# Release 409 (3 Mar 2023)
 
 ## General
 
 * Add support for dropping nested fields with a `DROP COLUMN` statement. ({issue}`15975`)
+* Improve output of `EXPLAIN` queries to show statistics when a query contains
+  aggregations. ({issue}`16201`)
+* Improve performance of queries with distinct aggregations using table
+  statistics. This can be configured with the `optimizer.mark-distinct-strategy`
+  configuration property, and the `optimizer.use-mark-distinct` configuration
+  property is now deprecated in favor of it. ({issue}`15927`)
 * Remove the `distributed-index-joins-enabled` configuration property and
   related functionality. ({issue}`15375`)
 * Fix failure when using non-comparable and non-sortable values as part of a
   `VALUES` expression. ({issue}`16242`)
 
 ## BigQuery connector
 
+* Add support for inserting default values. ({issue}`16327`)
```

Comment:
Honestly.. I am not sure what that means.. ?? 

Add support for using default values when inserting data